parser-typechecker/src/Unison/Builtin.hs

```diff
@@ -531,6 +532,7 @@ ioBuiltins =
   , ("IO.createDirectory.impl.v3", text --> iof unit)
   , ("IO.removeDirectory.impl.v3", text --> iof unit)
   , ("IO.renameDirectory.impl.v3", text --> text --> iof unit)
+  , ("IO.directoryContents.impl.v3", text --> iof (list text))
```

Comment:
Was directory contents stuff meant to be in this PR? I guess it'll probably go away once the other PR is merged?